test/JWT.hs

```diff
@@ -222,7 +241,7 @@ spec = do
       k = fromJust $ decode "{\"kty\":\"oct\",\"k\":\"\"}"
 
     describe "when the current time is prior to the Expiration Time" $
-      it "can be decoded and validated" $ do
+      it "can be decoded and validated" $
```

Comment:
please remove this (it is not relevant to this commit).

(feel free to submit a separate PR for such changes)